test/integration-test.py

```diff
@@ -265,15 +261,6 @@ def test_expiration_mailer():
     if mailcount != 2:
         raise Exception("\nExpiry mailer failed: expected 2 emails, got %d" % mailcount)
 
-def test_revoke_by_account():
```

Comment:
Why is test_revoke_by_account deleted? It seems useful.

Reply:
It was broken. It referenced vars that didn't exist & passed the wrong args to `revoke()`. When I fixed those low hanging fruit it started to fail because it relied on a cert file existing that wasn't there. I filed https://github.com/letsencrypt/boulder/issues/3518 as a follow up because while I agree that testing revocation with an authorized account is valuable the unused test we had for it has rotted to the point of needing rewriting.